press/targets/linux/debian/ubuntu_1604/ubuntu_1604_target.py

```diff
@@ -19,8 +19,16 @@ def check_for_grub(self):
                 raise GeneralPostTargetError(
                     'Error installing required packages for grub2')
 
+    def install_hwe_kernel_on_14gen(self):
```

Comment:
This really should be handled in the configuration. Perhaps adding a new option like `kernel_package`, or using the packages plugin.